src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1283,13 +1323,13 @@ private void UpdateImage(int key, PropertyValue value)
         /// Callback function to Lazy UpdateImage.
         /// Or, We can call UpdateImage() function directly if we need.
         /// </summary>
-        private void UpdateImage(object source, EventArgs e)
+        internal protected void UpdateImage(object source, EventArgs e)
```

Comment:
This method also can be opened with this protected internal.

Reply:
I think this API don't need to be protected. (This API only used for event callback). keep it as private.